docs/.vuepress/config.js

```diff
@@ -24,9 +24,10 @@ module.exports = {
         collapsable: false, // optional, defaults to true
         sidebarDepth: 1,    // optional, defaults to 1
         children: [
-          ['/releases', 'Releases'],
+          ['/latestrelease', 'Latest release'],
+          ['/previousreleases', 'Previous releases'],
```

Comment:
I prefer to have one big "Releases" entry (instead of Latest and Previous) and then organize "Release" with a subsection "Older Releases" pointing to each release.

Reply:
Done, renamed as "Current release" as discussed